bundles/org.openhab.binding.lgwebos/README.md

```diff
@@ -53,6 +53,33 @@ Please note that at least one channel must be bound to an item before the bindin
 | mediaPlayer     | Player    | Media control player                                                                                                                                                                                                    | W          |
 | mediaStop       | Switch    | Media control stop                                                                                                                                                                                                      | W          |
 | appLauncher     | String    | Application ID of currently running application. This also allows to start applications on the TV by sending a specific Application ID to this channel.                                                                 | RW         |
+| irCode          | String    | Simulates pressing of a button on the TV's infrared remote control. See below for a list of control strings.                                                                                                            | W          |
```

Comment:
irCode stands for Infra Red Code I suppose. Should we not name it Remote Control Button or sth similar. I am not sure if the remote actually uses infra red or another form of connection nowadays. The value which go over this channel certainly won't be the codes used by the remote internally.

Reply:
All these commands are sent to device as mouse buttons. Why not call it "button" instead?

Reply:
Well, it's already fixed. Sorry for the noise.
The fact that the PR has 80 commits and 50.000 changed lines makes it hard to review.